homeassistant/components/plugwise/strings.json

```diff
@@ -43,10 +43,10 @@
         "name": "Flame state"
       },
       "heating_state": {
-        "name": "[%key:component::climate::entity_component::_::state_attributes::hvac_action::state::heating%]"
```

Comment:
Are we sure we need this change?
![image](https://github.com/home-assistant/core/assets/26537646/645c4de4-3ec8-49f6-9e68-66934c701245)

It looks correct in lokalise

Reply:
These ones are correct, but we usually try to avoid referencing other platforms iirc

Reply:
We try to avoid them cross integrations; yes.

This is referencing core parts, which is fine 👍